### DISCUSSION TITLE
Feature Request: Adding option for time input as measurable

### DISCUSSION THREAD

**Discussion by kmsadaf** [iSoron/uhabits#1500] [Feature Requests]
Please add an option for time input as measurable, how long did you game today? i used 1.30 hours as 1.3 but it uses it as a decimal number, when i use 1.5 instead it becomes confusing seeing the numbers, so "time" should be added as an option besides "yes or no" and "measurable" or measurable option should have an sub option of hour unit

**Comment by herrbrixter** [comment]
Duplicate of #953
